.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,51 @@
+name: New issue for feature request
+description: "Have a feature suggestion? Share your idea with a feature request."
+title: "(Feature): <title>"
+labels: "type: feature"
+body:
+  # Greeter message
+  - type: markdown
+    attributes:
+        value: Thank you for suggesting a feature! Please first check the boxes below to ensure that your feature request meets our requirements.
+  # Requirements checks
+  - type: checkboxes
+    attributes:
+        options:
+            - label: I have read the [feature request guidelines](https://hackmd.io/@natron-dev-awesome/B1SW6Hbau)
+              required: true
+            - label: I have followed the [contributing guidelines](https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTING.md) to the best of my understanding
+              required: true
+            - label: This feature is not on the [issue tracker](https://github.com/NatronGitHub/Natron/issues?q=is%3Abug+type%3Afeature+type%3Aidea) and is not a duplicate
+              required: true
+  # Feature request description
+  - type: textarea
+    attributes:
+        label: Provide a description of your feature request
+        render: markdown
+    validations:
+        required: true
+  # Realistically implementable or not?
```

Comment:
Would be interested in how useful @devernay finds this section.

Reply:
Yeah - my original idea was to use this to categorize issues as either "features" or "ideas" (we have those two tags already). Not a must-have though.